### PR TITLE
Pass /etc/gce.conf to cluster autoscaler if needed

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @vulpecula @piosz @jszczepkowski @fgrzadkowski

### CODE REVIEW

File: cluster/gce/trusty/configure-helper.sh [hunk 1]
```diff
@@ -622,6 +622,12 @@ start_cluster_autoscaler() {
     remove_salt_config_comments "${src_file}"
 
     local params=`sed 's/^/"/;s/ /","/g;s/$/",/' <<< "${AUTOSCALER_MIG_CONFIG}"`
+    if [ -n "${PROJECT_ID:-}" ] && [ -n "${TOKEN_URL:-}" ] && [ -n "${TOKEN_BODY:-}" ] && [ -n "${NODE_NETWORK:-}" ]; then
+      params="${params} --cloud-config=/etc/gce.conf"
+    fi
+
+    sed -i -e "s@{{cloud_config_mount}}@${CLOUD_CONFIG_MOUNT}@g" "${src_file}"
```
Comment (piosz) [inline cluster/gce/trusty/configure-helper.sh]: Move those two lines after 631, to have the same order as above.

File: cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest [hunk 2]
```diff
@@ -1,5 +1,14 @@
 {% if pillar.get('enable_node_autoscaler', '').lower() == 'true' %}
-{% set params = pillar['autoscaler_mig_config'] -%}
+{% set cloud_config = "" -%}
+{% set cloud_config_mount = "" -%}
+{% set cloud_config_volume = "" -%}
+{% if grains.cloud == 'gce' and grains.cloud_config is defined -%}
```
Comment (piosz) [inline cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest]: Is cloud set to 'gce' in GKE? Or to 'gke'?

Comment (mwielgus) [inline cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest]: I copied it from controller manifest so I guess cloud is gce

File: cluster/gce/trusty/configure-helper.sh (hunk 1 above)
Comment (mwielgus) [inline cluster/gce/trusty/configure-helper.sh]: Done

**Comment by piosz** [comment]
LGTM

File: hack/verify-flags/exceptions.txt [hunk 3]
```diff
@@ -1,4 +1,87 @@
 Vagrantfile:      node_ip = $node_ips[n]
+_output/release-stage/full/kubernetes/Vagrantfile:      node_ip = $node_ips[n]
```
Comment (wojtek-t) [inline hack/verify-flags/exceptions.txt]: Please remove all those "_output" lines.

(You probably need to do make clean before running the flag script).

**Comment by wojtek-t** [comment]
Temporary removing lgtm - feel free to readd it after applying comment.

File: hack/verify-flags/exceptions.txt (hunk 3 above)
Comment (piosz) [inline hack/verify-flags/exceptions.txt]: Thanks @wojtek-t for catching this. As I wrote here https://github.com/kubernetes/kubernetes/pull/26572#discussion_r65225728 I have no idea what does it mean. Shouldn't this be caught by verify-all.sh script?

Comment (wojtek-t) [inline hack/verify-flags/exceptions.txt]: We probably should change the verify-flags script.

Comment (mwielgus) [inline hack/verify-flags/exceptions.txt]: Done.

**Pull request merged**